apps/examples/wifi_manager_sample/wm_test_stress_test1.c

```diff
@@ -146,22 +145,22 @@ TEST_SETUP(connect)
 {
 	ST_START_TEST;
 	ST_EXPECT_EQ(WIFI_MANAGER_SUCCESS, wifi_manager_init(&g_wifi_callbacks));
-	wifi_manager_ap_config_s apconfig;
-	wm_get_apinfo(&apconfig);
-	ST_EXPECT_EQ(WIFI_MANAGER_SUCCESS, wifi_manager_connect_ap(&apconfig));
 	ST_END_TEST;
 }
 
 TEST_TEARDOWN(connect)
 {
 	ST_START_TEST;
+	ST_EXPECT_EQ(WIFI_MANAGER_SUCCESS, wifi_manager_deinit());
 	ST_END_TEST;
 }
 
 TEST_F(connect)
 {
 	ST_START_TEST;
-	ST_EXPECT_EQ(WIFI_MANAGER_SUCCESS, wifi_manager_deinit());
+	wifi_manager_ap_config_s apconfig;
+	wm_get_apinfo(&apconfig);
+	ST_EXPECT_EQ(WIFI_MANAGER_SUCCESS, wifi_manager_connect_ap(&apconfig));	
```

Comment:
same as above

Reply:
purpose of connect, connect_done, disconnect and scan is to measure time of these respective operations.
Is it required to add deinit in TEST_F and add time of deinit execution in total TC execution time


> same as above

purpose of connect, connect_done, disconnect and scan is to measure time of these respective operations.
Is it required to add deinit in TEST_F and add time of deinit execution in total TC execution time

Reply:
> same as above

purpose of connect, connect_done, disconnect and scan is to measure time of these respective operations.
Is it required to add deinit in TEST_F and add time of deinit execution in total TC execution time?

Reply:
stress1 -> Connect_done ->
description: call deinit() after receiving connect done event

Originally code
TEST_SETUP -> init, connect ap,
TEST_F -> deinit
only deinit time was measured.

Modified code:
TEST_SETUP -> init,
TEST_F -> , connect ap
TEST_TEARDOWN -> deint
only connect_ap time is measured.

If required. we can add deinit in TEST_F.